syno-plex-update.sh

```diff
@@ -10,13 +10,20 @@
 
 # ========== [Begin Configuration] ==========
 
+DSM_VERSION=$(cat /etc/VERSION | grep majorversion | tail -c 3 | head -c 1)
+OS_ARCHITECTURE="linux-$(uname -m)"
+
 # Path to the local Plex server preferences file (in NAS filesystem).
 # Can be found fia `sudo find / -name "Preferences.xml" | grep Plex`
 # Extracting and passing the online token seems optional though, so you can omit this.
-PLEX_PREFERENCES_FILE='/volume1/@apphome/PlexMediaServer/Plex Media Server/Preferences.xml'
+if [ "${DSM_VERSION}" -ge "7" ]; then
+    PLEX_PREFERENCES_FILE='/var/packages/PlexMediaServer/shares/PlexMediaServer/AppData/Plex Media Server/Preferences.xml'
```

Comment:
Nice! I wasn't aware of the symlinked path; this definitely looks cleaner than `volume1`